mri/operators/fourier/utils.py

```diff
@@ -272,8 +272,8 @@ def get_stacks_fourier(kspace_loc):
     sort_pos = np.lexsort(tuple(kspace_loc[:, i]
                                 for i in np.arange(3)))
     kspace_loc = kspace_loc[sort_pos]
-    first_stack_len = np.size(np.where(kspace_loc[:, 2]
-                                       == np.min(kspace_loc[:, 2])))
+    first_stack_len = np.size(np.where(
+        kspace_loc[:, 2] == np.min(kspace_loc[:, 2])))
```

Comment:
Just as a nitpicking (don't need to correct here), the best way to avoid future unnecessary changes is :
```python
first_stack_len = np.size(np.where(
        kspace_loc[:, 2] == np.min(kspace_loc[:, 2]),
))
```

Reply:
Yes. Duly noted, this will change in future anyway.